src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexCharClass.cs

```diff
@@ -1543,7 +1543,11 @@ private static RegexCharClass ParseRecursive(string charClass, int start)
         public static string OneToStringClass(char c)
             => CharsToStringClass([c]);
 
-        internal static unsafe string CharsToStringClass(ReadOnlySpan<char> chars)
+        internal static
+#if !NET
+        unsafe
```

Comment:
This method is safe for callers. Move this `unsafe` into the method so that it wraps the unsafe code only?

Reply:
I ended up duplicating code since netfx needs unsafe context in two places